src/python/turicreate/toolkits/one_shot_object_detector/util/_augmentation.py

```diff
@@ -39,7 +39,12 @@ def preview_synthetic_training_data(data,
     out : SFrame
         An SFrame of sythetically generated annotated training data.
     """
+
     dataset_to_augment, image_column_name, target_column_name = check_one_shot_input(data, target)
+    original_dataset_size = len(dataset_to_augment)
+    dataset_to_augment = dataset_to_augment.dropna(columns=image_column_name)
+    if original_dataset_size != len(dataset_to_augment):
+        print("Dropping " + str(original_dataset_size - len(dataset_to_augment)) + " rows for missing features in the training dataset.")
```

Comment:
At this point it's already been dropped, so I would say "Dropped" rather than "Dropping". Also I think being specific is good: I would say "for images with values of None" rather than "for missing features".

Reply:
What do other toolkits print out? Where is this warning defined for those toolkits?

Reply:
@TobyRoseman @nickjong 
After discussions we have decided to make the toolkits consistent with what logistic classifier does in case missing values.